.github/workflows/lint-frontend.yml

```diff
@@ -23,4 +21,6 @@ jobs:
         run: npm install
 
       - name: Lint
-        run: npm run lint
\ No newline at end of file
+        run: |
+          npm run lint
+          npm run format
```

Comment:
`npm run format` will reformat the files.
I think we just want to check here instead? e.g. `npx prettier . --check`  or an equivalent script in package.json

Reply:
The reformat was intended, if we make it just a check, then we will probably need to add format in a pre-push or pre-commit. Is there any benefit in not just doing it actions?